frontend/public/components/utils/documentation.tsx

```diff
@@ -85,6 +85,11 @@ export const documentationURLs: documentationURLsType = {
     downstream: 'html/building_applications/deployments',
     upstream: 'applications/deployments/what-deployments-are.html',
   },
+  admissionWebhookWarning: {
+    downstream: '',
+    kube: `${KUBE_DOCS}/reference/access-authn-authz/extensible-admission-controllers/#response`,
+    upstream: '/reference/access-authn-authz/extensible-admission-controllers/#response',
```

Comment:
Should we use [https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)?

Reply:
> Should we use https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/?

@TheRealJon I think the suggestion will not work for `upstream` since its url prefix is `https://docs.okd.io/latest/`.
The OLM url seems to be the only doc source that has a link to the Admission Validating Webhook in   `https://docs.okd.io/latest/`. I think that is the closest to what we need for an `upstream` url. 
Also,the Admission Validating Webhooks are enabled using operators like getekeeper.
I think I will revert to my earlier change for this, and double check this later on with docs when the downstream url is available. Hope that is fine with everyone on this.

Reply:
See where the `operators/understanding/olm/olm-webhooks.html` link to: ![Screenshot 2024-04-30 at 2 54 39 PM](https://github.com/openshift/console/assets/15249132/f834fb7b-acd1-4925-8d77-d32c88160af9)

Reply:
I also searched around a bit and found this in the upstream and downstream docs:

Upstream: https://docs.okd.io/latest/architecture/index.html#about-admission-plug-ins
Downstream: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/architecture/admission-plug-ins

Reply:
> I also searched around a bit and found this in the upstream and downstream docs:
> 
> Upstream: https://docs.okd.io/latest/architecture/index.html#about-admission-plug-ins Downstream: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/architecture/admission-plug-ins

The downstream points to 4.15 and the admission plugin link  in the upstream link to 404 page.
However I will add them and get the most appropriate urls from docs team later on as promised by @opayne1
 I believe they know these things better than any of us.  
Let's close this issue and address the urls separately.